src/PointwiseFunctions/AnalyticSolutions/GrMhd/BondiMichel.hpp

```diff
@@ -23,42 +23,146 @@ namespace grmhd {
 namespace Solutions {
 
 /*!
- * \brief Bondi-Michel accretion with superposed magnetic field in Schwarzschild
- * spacetime in Kerr-Schild coordinates.
+ * \brief Bondi-Michel accretion \cite Michel1972 with superposed magnetic field
+ * in Schwarzschild spacetime in Cartesian Kerr-Schild coordinates.
  *
- * An analytic solution to the 3-D GrMhd system. The user specifies the sonic
+ * An analytic solution to the 3-D GRMHD system. The user specifies the sonic
  * radius \f$r_c\f$ and sonic rest mass density \f$\rho_c\f$, which are the
  * radius and rest mass density at the sonic point, the radius at which the
  * fluid's Eulerian velocity as seen by a distant observer overtakes the local
- * sound speed \f$c_{s,c}\f$. With a specified polytropic exponent \f$\Gamma\f$,
+ * sound speed \f$c_{s,c}\f$. With a specified polytropic exponent \f$\gamma\f$,
```

Comment:
In the documentation of the equations of state, I see:
- `IdealFluid.hpp`: gamma for the adiabatic index
- `PolytropicFluid.hpp`: Gamma for the polytropic exponent

Why make this change away from consistency?

Reply:
We need to move everything to the notation we are using in the paper (sounds like a fun task, right? :P ). So I figured I'd start here.